### PR TITLE
fix: update shuttle_live to assign initial shuttle for validate call, update seeds for FK duplication

### PR DESCRIPTION
**Pull request by meagharty**
#### Reviewer Checklist
- [ ] Meets ticket's acceptance criteria
- [ ] Any new or changed functions have typespecs
- [ ] Tests were added for any new functionality (don't just rely on Codecov)
- [ ] This branch was deployed to the staging environment and is currently running with no unexpected increase in warnings, and no errors or crashes.

### CODE REVIEW

File: lib/arrow_web/live/shuttle_live/shuttle_live.ex [hunk 1]
```diff
@@ -157,15 +157,16 @@ defmodule ArrowWeb.ShuttleViewLive do
 
   def handle_event("validate", %{"shuttle" => shuttle_params}, socket) do
     form =
-      %Shuttle{} |> Shuttles.change_shuttle(shuttle_params) |> to_form(action: :validate)
+      Shuttles.change_shuttle(socket.assigns.shuttle, shuttle_params)
+      |> to_form(action: :validate)
```
Comment (lemald) [inline lib/arrow_web/live/shuttle_live/shuttle_live.ex:161]: **nitpick:**
```suggestion
      socket.assigns.shuttle
      |> Shuttles.change_shuttle(shuttle_params)
      |> to_form(action: :validate)
```

File: lib/arrow_web/live/shuttle_live/shuttle_live.ex [hunk 2]
```diff
@@ -132,22 +132,22 @@ defmodule ArrowWeb.ShuttleViewLive do
...
-      })
+    shuttle = %Shuttle{
+      status: :draft,
+      routes: [%Shuttles.Route{direction_id: :"0"}, %Shuttles.Route{direction_id: :"1"}]
+    }
 
     gtfs_disruptable_routes = Shuttles.list_disruptable_routes()
     shapes = Shuttles.list_shapes()
-    form = to_form(changeset)
+    form = to_form(Shuttles.change_shuttle(shuttle))
```
Comment (lemald) [inline lib/arrow_web/live/shuttle_live/shuttle_live.ex:142]: **nitpick:**
```suggestion
    form = shuttle |> Shuttles.change_shuttle() |> to_form()
```

**Review by lemald — APPROVED** [review]
A couple of minor and very optional syntax nitpicks, but otherwise looks good!

File: lib/arrow_web/live/shuttle_live/shuttle_live.ex (hunk 1 above)
Comment (meagharty) [reply lib/arrow_web/live/shuttle_live/shuttle_live.ex:161]: 👍 I wish this would happen via format

**Pull request merged**